style.css

```diff
@@ -0,0 +1,21 @@
+
+
+#title {color: blue;}
```

Comment:
Please, take into account that the attribute should be listed independently in the next line after the opening bracket. The closing bracket should be standing by itself in the next line after the last attribute is declared . This is done to have a clean and easy to read code. This is common practice among web developers.